src/Compilers/Core/Portable/DiagnosticAnalyzer/AnalyzerDriver.cs

```diff
@@ -244,13 +244,6 @@ private void Initialize(AnalyzerExecutor analyzerExecutor, DiagnosticQueue diagn
                 }
             };
 
-            if (analysisOptions.LogAnalyzerExecutionTime)
```

Comment:
Did you try to run a compile with `/reportanalyzer` switch with third party analzyers (say FxCopAnalyzers)? Before this was added, the first analyzer to invoke `GetTypeByMetadataName` always took the biggest performance hit and significantly skewed the performance numbers, rendering this switch useless.

Reply:
I am not sure. this is here to avoid skewing analysis time. otherwise, we measure compilation/binding time not analyzer running time.